srcpkgs/i3blocks/template

```diff
@@ -7,7 +7,7 @@ make_build_args="SYSCONFDIR=/etc"
 hostmakedepends="autoconf automake pkg-config"
 makedepends="bash-completion"
 short_desc="Flexible scheduler for i3bar"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="André L. C. Moreira <alcm99@gmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/vivien/i3blocks"
 distfiles="${homepage}/archive/${version}.tar.gz"
```

Comment:
```suggestion
distfiles="https://github.com/vivien/i3blocks/archive/${version}.tar.gz"
```

Reply:
Thanks for your feedback. I'm not sure we should do that, because according to the [Manual](https://github.com/void-linux/void-packages/blob/master/Manual.md) such variable is mandatory in the template file.

Reply:
It's kind of the new (semi-)official convention to avoid using any variables other than `version` (or related variables) in `distfiles`. For example, see the commit description in https://git.vuxu.org/xtools/commit/?id=248be132dff32a1d2ec45dce0c84f878fe8fdc51:

> It also often doesn't save any characters and not using the variable
> makes prettier, simpler URLs that can often be copied from the template
> without manually replacing variables to get to the page with sources.

The commit message is referring to `${pkgname}`, but the same idea applies to `${homepage}` too.

Reply:
Thanks for the explanation. Suggestion implemented.